confidential-ledger-demo/server/confidential-setup.py

```diff
@@ -19,11 +22,15 @@
 
 #Set tenant ID from .env file
 
-TENANT_ID = os.getenv('AZURE_TENANT_ID')
-os.environ["AZURE_TENANT_ID"] = TENANT_ID
+clientId = os.getenv("CL_APP_ID")
+# scan:ignore
+clientSecret = os.getenv("CL_CLIENT_SC")
```

Comment:
you've got a mix of camelCase and snake_case variables - pylint should be flagging this @FergusKidd  http://pylint.pycqa.org/en/latest/user_guide/options.html#predefined-naming-styles